[DOCS-3558] Log rotation configurations (#14133)
DataDog/documentation: content/en/agent/guide/agent-log-files.md

  text: "Agent commands"
---

> The Datadog Agent does a logs rollover every 10MB by default. When a rollover occurs, one backup (`agent.log.1`) is kept. If a previous backup exists, it is overwritten during the rollover. You can control the maximum size of one log file and maximum amount of "old" log files to keep by setting `log_file_max_size`(default: 10485760), `log_file_max_rolls`(default: 1) in the [Agent main configuration file][1].

Reviewer: ```suggestion
The Datadog Agent does a logs rollover every 10MB by default. When a rollover occurs, one backup (`agent.log.1`) is kept. If a previous backup exists, it is overwritten during the rollover. You can set the maximum size of one log file and the maximum number of backup files to keep with `log_file_max_size`(default: 10485760 bytes) and `log_file_max_rolls`(default: 1) in the [Agent main configuration file][1].
```

> The Datadog Agent does a logs rollover every 10MB by default. When a rollover occurs, one backup (`agent.log.1`) is kept. If a previous backup exists, it is overwritten during the rollover. You can set the maximum size of one log file and the maximum number of backup files to keep with `log_file_max_size`(default: 10485760 bytes) and `log_file_max_rolls`(default: 1) in the [Agent main configuration file][1].

Reviewer: ```suggestion
The Datadog Agent does a logs rollover every 10MB by default. When a rollover occurs, one backup (`agent.log.1`) is kept. If a previous backup exists, it is overwritten during the rollover. To set the maximum size of one log file and the maximum number of backup files to keep, use `log_file_max_size`(default: 10485760 bytes) and `log_file_max_rolls`(default: 1) in the [Agent main configuration file][1].
```
